### PR TITLE
Document: changed README.md to include npm install for client

### PR DESCRIPTION
**Pull request by paulywill**
- [x] I have read [Chapter's contributing guidelines](https://github.com/freeCodeCamp/chapter/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update README.md`).
- [x] My pull request targets the `master` branch of Chapter.

Changed PR from @pdotsani suggestions in  #402 

This change includes an inline comment next to the `npm install` for a description.
Tested locally and inline comment is ignored.

### CODE REVIEW

**Comment by allella** [comment]
Thanks @paulywill is this change specific to any operating system or environment, or should we consider it an improvement for all situation?

**Comment by paulywill** [comment]
@allella It did resolve to start `docker-compose up` on my OSX.

However, I suspect it would be an improvement in all situations.

**Comment by allella** [comment]
@Zeko369 do you see any downside to this documentation change? 

We've had a couple contributors suggest a 

`cd client`
improved their installation.

File: README.md [hunk 1]
```diff
@@ -82,6 +82,8 @@ cd chapter
 Install dependencies:
 ```
 npm install
+cd client
```
Comment (allella) [inline README.md:85]: Not sure it matters, but tomn on Discord used `cd ./client`

As far as I know they are the equivalent, but if there's an OS-specific reason to include the period, then we could consider it.

Comment (paulywill) [reply README.md]: Agree. Made the suggested change.

**Review by timmyichen — COMMENTED** [review]
Thanks for adding this! Do you think it's more semantically clear if we just install things recursively, e.g. with:

```
npx recursive-install
```

Thoughts?

**Comment by paulywill** [comment]
Wow! I had no idea that command existed.

**Comment by paulywill** [comment]
@timmyichen I have tested locally and it appears to accomplish the same thing... always allows flexibility and not hardcoding to structure of directories (a minor concern but valid).

I will make the change to the PR.

**Review by timmyichen — APPROVED** [review]
:tada

**Comment by allella** [comment]
Nice. Teamwork makes the dream work.

**Comment by allella** [comment]
@all-contributors please add @paulywill for docs.

**Comment by tomnoland** [comment]
Didn't know about that either. Looks good to me

**Pull request merged**